src/chi/components/tables/tables.scss

```diff
@@ -134,6 +134,71 @@ th {
     }
   }
 
+  &.-portal {
+    caption {
+      background-color: set-color(grey, 30);
+      font-weight: 700;
+      padding-bottom: 0.75rem;
+      padding-left: 1rem;
+      padding-top: 0.75rem;
+      text-align: left;
+    }
+
+    thead {
+      tr {
+        th {
+          background-color: set-color(grey, 25);
+          border-bottom: 0;
+          font-weight: 700;
+          padding-bottom: 0.75rem;
+          padding-top: 0.75rem;
+
+          &:nth-child(2) {
+            text-align: right;
+          }
+        }
+      }
+    }
+
+    tbody {
+      tr {
+        &:nth-child(odd) {
+          background-color: set-color(grey, 20);
+        }
+
+        &:hover {
+          & > td {
+            background-color: rgb(205, 219, 240);
+
+            &:first-child {
+              border-left: 0.25rem solid $primary-color;
+              padding-left: 0.75rem;
+            }
+          }
+        }
+
+        td {
+          border-bottom: 0;
+          padding-bottom: 0.75rem;
+          padding-top: 0.75rem;
```

Comment:
Let's remove the padding defined here. All base tables should have the same font and row size, portal included. If portal needs to render smaller or larger tables, they can use table size classes. The `-xs` class will allow them to achieve 32px row heights.